app/points/calculate-points-for-tier.js

```diff
@@ -2,8 +2,26 @@ const TierCounts = require('../../app/points/domain/tier-counts')
 const assertObjectType = require('../../app/points/domain/validation/assert-object-type')
 const assertNumber = require('../../app/points/domain/validation/assert-number')
 
-var calculateWeightedPoints = function (count, points, weighting = 1) {
-  return count * (points * weighting)
+var calculateWeightedPoints = function (count, points, weighting) {
+  var weightedPoints = points
+
+  if (weighting !== undefined) {
+    weightedPoints = points * invertWeightingPercentage(weighting)
+  }
+
+  return count * weightedPoints
+}
+
+var invertWeightingPercentage = function (weightingPercentage) {
+  var multiplier
+  if (weightingPercentage >= 100) {
+    multiplier = 0
```

Comment:
When this is returned the the weightedPoints will always be 0. Is the intended purpose that if the weighting is above 100% the points are reset to 0?

Reply:
Yea, that's the current functionality of WMT.